mint.sh

```diff
@@ -140,6 +140,10 @@ function main()
     export ENABLE_VIRTUAL_STYLE
     export RUN_ON_FAIL
 
+    # Set to '1' if mc tests should get executed only with an sse-s3 encrypted bucket.
```

Comment:
'Only' is a bit confusing to me.

Are we only using one bucket? Are we only executing specific encryption tests?

I'm not 100% sure, but I think that all that happens is that encryption is activated on any buckets created for the test. If that is true, IMO the sentence becomes clearer if you remove the 'only'. 🙂

Reply:
Done